requirements/requirements.in

```diff
@@ -10,20 +10,20 @@ cryptography>=39.0.1 ## https://github.com/ansible/awx/security/dependabot/90
 Cython<3 # Since the bump to PyYAML 5.4.1 this is now a mandatory dep
 daphne
 distro
-django==3.2.16  # see UPGRADE BLOCKERs https://github.com/ansible/awx/security/dependabot/67
+django==4.2  # see UPGRADE BLOCKERs
 django-auth-ldap
 django-cors-headers
 django-crum
 django-extensions
 django-guid==3.2.1
-django-oauth-toolkit==1.4.1
+django-oauth-toolkit<2.0.0      # Version 2.0.0 has breaking changes that will need to be worked out before upgrading
```

Comment:
That is correct. But it should grab the latest compatable version when we run the upgrade script `requirements/updater.sh` and that then produces the requirements.txt file which would have a fixed pinned version:
```
django-oauth-toolkit==1.7.1
```

Reply:
OK. Cool. I thought it might be something like that when I saw the `.in` on it. I'm not in love with that workflow (or that `updater.sh` script since it claims to be `sh` but has bashisms in it), but I'm sure there's a good reason for it that I don't understand yet.